schema.sql

```diff
@@ -8,5 +8,6 @@ CREATE TABLE animals (
   escape_attempts INT,
   neutered BOOLEAN,
   weight_kg DECIMAL,
+  species VARCHAR(40),
```

Comment:
- [ Required ] I think it would be better if you used the `ALTER TABLE` SQL command to change the structure of the `animals` table. In a real-life situation, it would be difficult to add a column the way you have done above since the table would have already been created.

## Example code
```SQL
  ALTER TABLE animals ADD species varchar(255);
```